ext/lib/crypto/mbedtls/Kconfig

```diff
@@ -98,3 +98,12 @@ config MBEDTLS_HEAP_SIZE
 	  application. For server application 15000 bytes should be enough.
 	  For some dedicated and specific usage of mbedtls API, the 1000 bytes
 	  might be ok.
+
+config APP_LINK_WITH_MBEDTLS
```

Comment:
I do not see where this guy is bring used

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/5498/files#diff-b9537eb0f71fef9b0fdb1e934a042cecR272